README.md

```diff
@@ -24,6 +24,7 @@ language.
 * [Pass variables to closure](idioms/pass-var-to-closure.md)
 * [`mem::replace(_)` to avoid needless clones](idioms/mem-replace.md)
 * [Temporary mutability](idioms/temporary-mutability.md)
+* [Out of the Box Dynamic Dispatch](idioms/ootb-dyn-dispatch.md)
```

Comment:
Isn't this title now out of sync with the title below?

Reply:
Right, fixed.